readme.md

```diff
@@ -108,6 +108,15 @@ Shortcut for sending JSON. Use this instead of the `body` option. Accepts a plai
 
 Sets `options.method` to the method name and makes a request.
 
+#### baseUrl
+
+Type: `string` [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL)
+
+When specified, `url` will be prepended by `baseUrl`.<br>
+If you specify an absolute URL, it will skip the `baseUrl`.
```

Comment:
Can you show examples of valid uses, like done in the Got docs: https://github.com/sindresorhus/got#baseurl You can just adapt those.